com.unity.render-pipelines.universal/Runtime/UniversalAdditionalCameraData.cs

```diff
@@ -63,6 +63,16 @@ public enum AntialiasingQuality
         High
     }
 
+    /// <summary>
+    /// Controls the update frequency for the Volume Framework
```

Comment:
The enum itself does not control anything. Oleksandr can probably come up with a better suggestion.
```suggestion
    /// Defines the update frequency for the Volume Framework.
```
Do we also need descriptions for the values in the enum? It would be valuable to describe how the user can control updates when ViaScripting is set.

Reply:
Removed this enum as we wanted to only have one with this feature.